test/run-drun/stable-counter-class/stable-counter-class-2.mo

```diff
@@ -0,0 +1,24 @@
+import Prim "mo:⛔";
+
+persistent actor class Counter() {
+
+  var count : Nat = 0;
```

Comment:
Maybe we should one more test with `transient`, even if uninteresting for testing, just for syntax coverage.